### ISSUE TITLE
Mój pomysł

### ISSUE THREAD

**Issue by MaksDradrach** [szczepienia/szczepienia.github.io#47]
Według mnie super byłoby np. wyszukiwanie terminu według kodu pocztowego albo lokalizacji, żeby znaleźć ten najbliżej.

**Comment by AdrianRamzes** [comment]
> Według mnie super byłoby np. wyszukiwanie terminu według kodu pocztowego albo lokalizacji, żeby znaleźć ten najbliżej.

Z mieć tytuł tego issue i typ na feature request

**Comment by szczepienia** [comment]
@MaksDradrach Tak, nad tym pracuję (tylko problemy z dostępem trochę mi to opóźniły) w ramach https://github.com/szczepienia/szczepienia.github.io/issues/17
